core/components/atoms/dropdown/option/IconOption.tsx

```diff
@@ -25,9 +24,7 @@ const IconOption = (props: OptionTypeProps) => {
       data-disabled={disabled}
     >
       {/* eslint-enable  */}
-      {icon && (
-        <Icon className="Option-icon mr-4" data-test={`${dataTest}--Icon`} name={icon} appearance={appearance} />
-      )}
+      {icon && <Icon className="Option-icon mr-4" data-test={`${dataTest}--Icon`} name={icon} />}
```

Comment:
Please do not remove appearance but just override the active for selected in CSS.